### ISSUE TITLE
Blocked with Henkaku

### ISSUE THREAD

**Issue by BenMitnicK** [joel16/Ultimate-VSH-Menu-PSV#1]
I tried the plugin with mode * ALL and I found myself stuck. I activate the plugin and when I launched App I have error message so I reboot the vita now Every time I try to launch Henkaku the console shutdown so I think I must to format the MC

**Comment by joel16** [comment]
I don't think you need to format the MC. I think if you keep holding the R button there's a way to reload henakaku and disable all plugins. Once you've done that you can open config.txt from vitashell and you can comment out the *ALL bit, and reload taihen cache using molecular shell.
Also next time, please be more cautious when a plugin is indicated as a 'pre-release' and a test build, I would recommend against using *ALL until this matures.

**Comment by BenMitnicK** [comment]
I know the risks I am the only one to trace my experience. I will test to boot with R trigger

**Comment by joel16** [comment]
Alright were you able to comment the *ALL out and reload tai cache?

**Comment by BenMitnicK** [comment]
With R Trigger he handed the config to zero and he installed me Henkaku as if it was the first time :)

**Comment by joel16** [comment]
Sweet, just glad you didn't have to format your MC or anything.

**Comment by BenMitnicK** [comment]
It is not a worry at least the info is up :)
